apps/src/lab2/views/components/Instructions.tsx

```diff
@@ -106,8 +115,12 @@ interface InstructionsPanelProps {
   messageIndex?: number;
   /** Optional image URL to display. */
   imageUrl?: string;
-  /** If the next button should be shown. */
-  showNextButton?: boolean;
```

Comment:
Nit: optional suggestion for this interface & in general - IMO it's preferrable to give a variable/param a more descriptive name rather than a short name with extra description in a comment. That way you know the full meaning wherever that var is used as you're reading the code, and it's less likely to get out of date as the code changes (easier to forget to update the comment than the var name).

some examples: 
`text` -> `primaryDisplayText`
`showContinueButton` -> `shouldShowContinueButton`
`showFinishButton` -> `shouldShowFinishButton`
`beforeNextLevel` -> `onBeforeNextLevelNavigation` 
`onNextPanel` -> `onNextButtonClick`

These name changes would cause the extra comments to be redundant and they could be removed.

Reply:
Thanks! I probably lean towards longer descriptive names, but I've gotten feedback in the past to shorten variable names. So I think I'll keep as is for now, but definitely worth a team discussion!